benchalerts/clients.py

```diff
@@ -179,7 +180,7 @@ def update_commit_status(
 
         json = {
             "state": state.value,
-            "description": description,
+            "description": textwrap.shorten(description, 140),
```

Comment:
This was another bug in the same Velox run; the description can't be more than 140 chars.